import React, { Component } from 'react';
import ReactDom from 'react-dom';
// import s from '../assets/mystyles.css';

class introduction_to_crispr extends Component {
    render() {
        return (
            <div>
                <h1> CRISPR </h1>
                <hr color="#665451" />
                <br />
             <div class="video-container" id="noob">
                    <iframe width="700" height="500" src="//www.youtube.com/embed/0LSnQukLmM4" frameborder="10" allowfullscreen></iframe>
                </div>
                <div>
                    <p>
                        The discovery of clustered DNA repeats began independently in three parts of the world.
                    </p>

                    <p>
                        The first description of what would later be called CRISPR was from Osaka University
researcher Yoshizumi Ishino in 1987, who accidentally cloned part of a CRISPR together
with the iap gene, the target of interest. The organization of the repeats was unusual
because repeated sequences are typically arranged consecutively along DNA. The function
of the interrupted clustered repeats was not known at the time.[18][19]
                    </p>
                        
                    <p>
                         In 1993 researchers of Mycobacterium tuberculosis in the Netherlands published two arti-
cles about a cluster of interrupted direct repeats (DR) in this bacterium. These researchers
recognized the diversity of the DR-intervening sequences among different strains of M. tuber-
culosis[20] and used this property to design a typing method that was named Spoligotyping,
which is still in use today.[21][22]               
                    </p>

                    <p>
                        At the same time, repeats were observed in the archaeal organisms Haloferax and Haloarcula
species, and their function was studied by Francisco Mojica at the University of Alicante in
Spain. Although his hypothesis turned out to be wrong, Mojica surmised at the time that
the clustered repeats had a role in correctly segregating replicated DNA into daughter cells
during cell division because plasmids and chromosomes with identical repeat arrays could
not coexist in Haloferax volcanii. Transcription of the interrupted repeats was also noted
for the first time.[22][23] By 2000, Mojica’s group had identified interrupted repeats in 20
species of microbes.[24] In 2001, Mojica and Ruud Jansen, who was searching for additional
interrupted repeats, proposed the acronym CRISPR (Clustered Regularly Interspaced Short
Palindromic Repeats) to alleviate the confusion stemming from the numerous acronyms used
to describe the sequences in the scientific literature.[23][25]
                    </p>

                    <p>
                        The first description of what might later be called CRISPR was from Osaka University
specialist Yoshizumi Ishino in 1987, who inadvertently cloned some portion of a CRISPR
together with the iap gene, the target of interest. The association of the rehashes was irreg-
ular in light of the fact that rehashed successions are ordinarily masterminded successively
along DNA. The capacity of the interfered with bunched rehashes was not known at the
time.[18][19]
                    </p>

                    <p>
                        *In 1993 analysts of Mycobacterium tuberculosis in the Netherlands distributed two articles
about a bunch of intruded on direct rehashes (DR) in this bacterium. These scientists
perceived the assorted qualities of the DR-interceding arrangements among various strains
of M. tuberculosis[20] and utilized this property to plan a writing strategy that was named
Spoligotyping, which is still being used today.[21][22]
                    </p>

                    <p>
                        *In the meantime, rehashes were seen in the archaeal living beings Haloferax and Haloarcula
species, and their capacity was considered by Francisco Mojica at the University of Alicante
in Spain. Despite the fact that his theory ended up being incorrectly, Mojica inferred at the
time that the bunched rehashes had a part in accurately isolating duplicated DNA into girl
cells amid cell division since plasmids and chromosomes with indistinguishable rehash clusters
couldn’t exist together in Haloferax volcanii. Translation of the interfered with rehashes was
likewise noted for the principal time.[22][23] By 2000, Mojica’s gathering had recognized
intruded on rehashes in 20 types of microbes.[24] In 2001, Mojica and Ruud Jansen, who
was looking for extra interfered with rehashes, proposed the acronym CRISPR (Clustered
Regularly Interspaced Short Palindromic Repeats) to reduce the perplexity originating from
the various acronyms used to depict the groupings in the logical literature.[23][25]
                    </p>

                    <img src="assets/index.jpg"/>

                    <p>
                        <b>Figure 1:</b> Simplified diagram of a CRISPR locus. The three major components of a CRISPR
locus are shown: cas genes, a leader sequence, and a repeat-spacer array. Repeats are shown
as gray boxes and spacers are colored bars. The arrangement of the three components is not
always as shown.[1][5] In addition, several CRISPRs with similar sequences can be present
in a single genome, only one of which is associated with cas genes.[4])
                    </p>

                    <h4> Mechanism </h4>
                    <p>
                        CRISPR-Cas immunity is a natural process of bacteria and archaea. CRISPR-Cas prevents
bacteriophage infection, conjugation and natural transformation by degrading foreign nucleic
acids that enter the cell.[33]
                    </p>

                    <p>
                        *CRISPR-Cas invulnerability is a characteristic procedure of microscopic organisms and
archaea. CRISPR-Cas counteracts bacteriophage disease, conjugation and common change
by corrupting remote nucleic acids that enter the cell.[33]
                    </p>

                    <h4> Spacer acquisition </h4>

                    <p>
                        When a microbe is invaded by a virus, the first stage of the immune response is to capture
viral DNA and insert it into a CRISPR locus in the form of a spacer. Cas1 and Cas2 are
found in all three types of CRISPR-Cas immune systems, which indicates that they are
involved in spacer acquisition. Mutation studies confirmed this hypothesis, showing that
removal of cas1 or cas2 stopped sp acer acquisition, without affecting CRISPR immune
response.[70][71][72][73][74]
                    </p>
                    <p>
                        Multiple Cas1 proteins have been characterised and their structures resolved.[75][76][77] Cas1
proteins have diverse amino acid sequences. However, their crystal structures are similar and
all purified Cas1 proteins are metal-dependent nucleases/integrases that bind to DNA in
a sequence-independent manner.[60] Representative Cas2 proteins have been characterised
and possess either (single strand) ssRNA-[78] or (double strand) dsDNA-[79][80] specific
endoribonuclease activity.
                    </p>
                    <p>
                        In the I-E system of E. coli Cas1 and Cas2 form a complex where a Cas2 dimer bridges two
Cas1 dimers.[81] In this complex Cas2 performs a non-enzymatic scaffolding role,[81] binding
double-stranded fragments of invading DNA, while Cas1 binds the single-stranded flanks of
the DNA and catalyses their integration into CRISPR arrays.[82][83][84]
                    </p>
                    <p>
                        *At the point when a microorganism is attacked by an infection, the primary phase of the
safe reaction is to catch viral DNA and embed it into a CRISPR locus as a spacer. Cas1
and Cas2 are found in every one of the three sorts of CRISPR-Cas invulnerable frameworks,
which demonstrates that they are included in spacer obtaining. Transformation concentrates
affirmed this theory, demonstrating that evacuation of cas1 or cas2 ceased spacer securing,
without influencing CRISPR invulnerable response.[70][71][72][73][74]
                    </p>
                    <p>
                        *Different Cas1 proteins have been portrayed and their structures resolved.[75][76][77] Cas1
proteins have assorted amino corrosive successions. Nonetheless, their precious stone struc-
tures are comparative and all sanitized Cas1 proteins are metal-subordinate nucleases/inte-
grases that predicament to DNA in an arrangement free manner.[60] Representative Cas2
proteins have been described and have either (single strand) ssRNA-[78] or (twofold strand)
dsDNA-[79][80] particular endoribonuclease action.
                    </p>

                    <p>
                        *In the I-E arrangement of E. coli Cas1 and Cas2 frame a perplexing where a Cas2 dimer
spans two Cas1 dimers.[81] In this complex Cas2 plays out a non-enzymatic platform role,[81]
restricting twofold stranded pieces of attacking DNA, while Cas1 ties the single-stranded
flanks of the DNA and catalyzes their joining into CRISPR arrays.[82][83][84]
                    </p>

                   <img src="assets/a.png" height="800px" width="800px"/>
                    <p>
                        <b>Figure 2:</b> The stages of CRISPR immunity for each of the three major types of adaptive
immunity. (1) Acquisition begins by recognition of invading DNA by Cas1 and Cas2 and
cleavage of a protospacer. (2) The protospacer is ligated to the direct repeat adjacent to
the leader sequence and (3) single strand extension repairs the CRISPR and duplicates the
direct repeat. The crRNA processing and interference stages occur differently in each of the
three major CRISPR systems. (4) The primary CRISPR transcript is cleaved by cas genes
to produce crRNAs. (5) In type I systems Cas6e/Cas6f cleave at the junction of ssRNA and
dsRNA formed by hairpin loops in the direct repeat. Type II systems use a trans-activating
(tracr) RNA to form dsRNA, which is cleaved by Cas9 and RNaseIII. Type III systems
use a Cas6 homolog that does not require hairpin loops in the direct repeat for cleavage.
(6) In type II and type III systems secondary trimming is performed at either the 5’ or 3’
end to produce mature crRNAs. (7) Mature crRNAs associate with Cas proteins to form
interference complexes. (8) In type I and type II systems, interactions between the protein
and PAM sequence are required for degradation of invading DNA. Type III systems do
not require a PAM for successful degradation and in type III-A systems basepairing occurs
between the crRNA and mRNA rather than the DNA, targeted by type III-B systems.[3])
                    </p>

                    <h4>Protospacer adjacent motifs</h4>
                    <p>
                        Bioinformatic analysis of regions of phage genomes that were excised as spacers (termed
protospacers) revealed that they were not randomly selected but instead were found adjacent
to short (3 – 5 bp) DNA sequences termed protospacer adjacent motifs (PAM). Analysis of
CRISPR-Cas systems showed PAMs to be important for type I and type II, but not type III
systems during acquisition.[29][85][86][87][88][89] In type I and type II systems, protospacers
are excised at positions adjacent to a PAM sequence, with the other end of the spacer cut
using a ruler mechanism, thus maintaining the regularity of the spacer size in the CRISPR
array.[90][91] The conservation of the PAM sequence differs between CRISPR-Cas systems
and appears to be evolutionarily linked to Cas1 and the leader sequence.[89][92]
                    </p>
                    <p>
                            New spacers are added to a CRISPR array in a directional manner,[27] occurring preferen-
tially,[56][85][86][93][94] but not exclusively, adjacent[88][91] to the leader sequence. Analysis
of the type I-E system from E. coli demonstrated that the first direct repeat adjacent to the
leader sequence, is copied, with the newly acquired spacer inserted between the first and
second direct repeats.[73][90]
                    </p>
                    <p>
                        The PAM sequence appears to be important during spacer insertion in type I-E systems.
That sequence contains a strongly conserved final nucleotide (nt) adjacent to the first nt of
the protospacer. This nt becomes the final base in the first direct repeat.[74][95][96] This
suggests that the spacer acquisition machinery generates single-stranded overhangs in the
second-to-last position of the direct repeat and in the PAM during spacer insertion. However,
not all CRISPR-Cas systems appear to share this mechanism as PAMs in other organisms
do not show the same level of conservation in the final position.[92] It is likely that in those
systems, a blunt end is generated at the very end of the direct repeat and the protospacer
during acquisition.
                    </p>
                    <p>
                        *Bioinformatic investigation of districts of phage genomes that were extracted as spacers
(named protospacers) uncovered that they were not haphazardly chosen but rather were
discovered neighboring short (3 – 5 bp) DNA groupings named protospacer adjoining themes
(PAM). Investigation of CRISPR-Cas frameworks demonstrated PAMs to be imperative for
sort I and sort II, yet not sort III frameworks amid acquisition.[29][85][86][87][88][89] In sort
I and sort II frameworks, protospacers are extracted at positions nearby a PAM grouping,
with the flip side of the spacer cut utilizing a ruler instrument, along these lines keeping
up the normality of the spacer estimate in the CRISPR array.[90][91] The protection of the
PAM arrangement varies between CRISPR-Cas frameworks and seems, by all accounts, to
be developmentally connected to Cas1 and the pioneer sequence.[89][92]
                    </p>
                    <p>
                        *New spacers are added to a CRISPR exhibit in a directional manner,[27] happening prefer-
entially,[56][85][86][93][94] yet not solely, adjacent[88][91] to the pioneer grouping. Examina-
tion of the sort I-E framework from E. coli exhibited that the main direct rehash neighboring
the pioneer arrangement, is replicated, with the recently obtained spacer embedded between
the first and second direct repeats.[73][90]
                    </p>
                    <p>
                        *The PAM arrangement has all the earmarks of being critical amid spacer addition in sort
I-E frameworks. That grouping contains an unequivocally moderated last nucleotide (nt)
adjoining the primary nt of the protospacer. This nt turns into the last base in the main
direct repeat.[74][95][96] This proposes the spacer securing apparatus creates single-stranded
shades in the second-to-last position of the immediate rehash and in the PAM amid spacer
inclusion. In any case, not all CRISPR-Cas frameworks seem to share this component as
PAMs in different living beings don’t demonstrate a similar level of preservation in the last
position.[92] It is likely that in those frameworks, a limit end is produced at the very end of
the immediate rehash and the protospacer amid obtaining.
                    </p>

                    <h4>
                        Insertion variants
                    </h4>
                    <p>
                        Analysis of Sulfolobus solfataricus CRISPRs revealed further complexities to the canonical
model of spacer insertion, as one of its six CRISPR loci inserted new spacers randomly
throughout its CRISPR array, as opposed to inserting closest to the leader sequence.[91]
                    </p>
                    <p>
                        Multiple CRISPRs contain many spacers to the same phage. The mechanism that causes
this phenomenon was discovered in the type I-E system of E. coli. A significant enhancement
in spacer acquisition was detected where spacers already target the phage, even mismatches
to the protospacer. This ‘priming’ requires the Cas proteins involved in both acquisition
and interference to interact with each other. Newly acquired spacers that result from the
priming mechanism are always found on the same strand as the priming spacer.[74][95][96]
This observation led to the hypothesis that the acquisition machinery slides along the foreign
DNA after priming to find a new protospacer.[96]
                    </p>
                    <p>
                        *Investigation of Sulfolobus solfataricus CRISPRs uncovered further complexities to the
accepted model of spacer inclusion, as one of its six CRISPR loci embedded new spacers
haphazardly all through its CRISPR cluster, rather than embeddings nearest to the pioneer
sequence.[91]
                    </p>
                    <p>
                        *Different CRISPRs contain numerous spacers to a similar phage. The component that
causes this wonder was found in the sort I-E arrangement of E. coli. A critical improvement
in spacer securing was recognized where spacers as of now focus on the phage, even confuses
to the protospacer. This ”preparing” requires the Cas proteins required in both securing
and impedance to collaborate with each other. Recently gained spacers that outcome from
the preparing system are constantly found on an indistinguishable strand from the preparing
spacer.[74][95][96] This perception prompted the speculation that the procurement apparatus
slides along the remote DNA subsequent to preparing to locate another protospacer.[96]
                    </p>

                    <h4>
                        Biogenesis
                    </h4>
                    <p>
                        CRISPR-RNA (crRNA), which later guides the Cas nuclease to the target during the in-
terference step, must be generated from the CRISPR sequence. The crRNA is initially
transcribed as part of a single long transcript encompassing much of the CRISPR array.[5]
This transcript is then cleaved by Cas proteins to form crRNAs. The mechanism to produce
crRNAs differs among CRISPR-Cas systems. In type I-E and type I-F systems, the pro-
teins Cas6e and Cas6f respectively, recognise stem-loops[97][98][99] created by the pairing of
identical repeats that flank the crRNA.[100] These Cas proteins cleave the longer transcript
at the edge of the paired region, leaving a single crRNA along with a small remnant of the
paired repeat region.
                    </p>
                    <p>
                        Type III systems also use Cas6, however their repeats do not produce stem-loops. Cleavage
instead occurs by the longer transcript wrapping around the Cas6 to allow cleavage just
upstream of the repeat sequence.[101][102][103]
                    </p>
                    <p>
                        Type II systems lack the Cas6 gene and instead utilize RNaseIII for cleavage. Functional
type II systems encode an extra small RNA that is complementary to the repeat sequence,
known as a trans-activating crRNA (tracrRNA).[71] Transcription of the tracrRNA and the
primary CRISPR transcript results in base pairing and the formation of dsRNA at the repeat
sequence, which is subsequently targeted by RNaseIII to produce crRNAs. Unlike the other
two systems the crRNA does not contain the full spacer, which is instead truncated at one
end.[66]
                    </p>
                    <p>
                        CrRNAs associate with Cas proteins to form ribonucleotide complexes that recognize foreign
nucleic acids. CrRNAs show no preference between the coding and non-coding strands, which
is indicative of an RNA-guided DNA-targeting system.[8][35][70][74][104][105][106] The type
I-E complex (commonly referred to as Cascade) requires five Cas proteins bound to a single
crRNA.[107][108]
                    </p>
                    
                    <p>
                    *CRISPR-RNA (crRNA), which later aides the Cas nuclease to the objective amid the
obstruction step, must be created from the CRISPR arrangement. The crRNA is at first
deciphered as a major aspect of a solitary long transcript incorporating a great part of the
CRISPR array.[5] This transcript is then divided by Cas proteins to shape crRNAs. The
component to create crRNAs varies among CRISPR-Cas frameworks. In sort I-E and sort I-
F frameworks, the proteins Cas6e and Cas6f separately, perceive stem-loops[97][98][99] made
by the blending of indistinguishable rehashes that flank the crRNA.[100] These Cas proteins
sever the more drawn out transcript at the edge of the combined locale, leaving a solitary
crRNA alongside a little remainder of the matched rehash area.
                    </p>
                    <p>
                        *Sort III frameworks likewise utilize Cas6, however their rehashes don’t create stem-circles.
Cleavage rather happens by the more extended transcript wrapping around the Cas6 to
permit cleavage only upstream of the rehash sequence.[101][102][103]
                    </p>
                    <p>
                    *Sort II frameworks do not have the Cas6 quality and rather use RNaseIII for cleavage.
Utilitarian sort II frameworks encode an additional little RNA that is integral to the rehash
arrangement, known as a trans-initiating crRNA (tracrRNA).[71] Transcription of the tracr-
RNA and the essential CRISPR transcript brings about base matching and the development
of dsRNA at the rehash succession, which is in this manner focused by RNaseIII to deliver
crRNAs. Not at all like the other two frameworks the crRNA does not contain the full
spacer, which is rather truncated at one end.[66]
                    </p>
                    <p>
                        *CrRNAs connect with Cas proteins to shape ribonucleotide buildings that perceive re-
mote nucleic acids. CrRNAs demonstrate no inclination between the coding and non-coding
strands, which is characteristic of a RNA-guided DNA-focusing on system.[8][35][70][74][104][105][106]
The sort I-E complex (ordinarily alluded to as Cascade) requires five Cas proteins bound to
a solitary crRNA.[107][108]
                    </p>
                    <h4> Interference </h4>
                    <p>
                        During the interference stage in type I systems the PAM sequence is recognized on the
crRNA-complementary strand and is required along with crRNA annealing. In type I systems
correct base pairing between the crRNA and the protospacer signals a conformational change
in Cascade that recruits Cas3 for DNA degradation.
                    </p>
                    <p>
                        Type II systems rely on a single multifunctional protein, Cas9, for the interference step.[66]
Cas9 requires both the crRNA and the tracrRNA to function and cleaves DNA using its
dual HNH and RuvC/RNaseH-like endonuclease domains. Basepairing between the PAM
and the phage genome is required in type II systems. However, the PAM is recognized on
the same strand as the crRNA (the opposite strand to type I systems).
                    </p>
                    <p>
                        Type III systems, like type I require six or seven Cas proteins binding to crRNAs.[109][110]
The type III systems analysed from S. solfataricus and P. furiosus both target the mRNA
of phages rather than phage DNA genome,[61][110] which may make these systems uniquely
capable of targeting RNA-based phage genomes.[60]
                    </p>
                    <p>
                        The mechanism for distinguishing self from foreign DNA during interference is built into the
crRNAs and is therefore likely common to all three systems. Throughout the distinctive
maturation process of each major type, all crRNAs contain a spacer sequence and some
portion of the repeat at one or both ends. It is the partial repeat sequence that prevents
the CRISPR-Cas system from targeting the chromosome as base pairing beyond the spacer
sequence signals self and prevents DNA cleavage.[111] RNA-guided CRISPR enzymes are
classified as type V restriction enzymes.
                    </p>
                    <p>
                        *Amid the impedance arrange in sort I frameworks the PAM grouping is perceived on the
crRNA-integral strand and is required alongside crRNA toughening. In sort I frameworks rec-
tify base blending between the crRNA and the protospacer signals a conformational change
in Cascade that enlisted people Cas3 for DNA debasement.
                    </p>
                    <p>
                        *Sort II frameworks depend on a solitary multifunctional protein, Cas9, for the impedance
step.[66] Cas9 requires both the crRNA and the tracrRNA to work and cuts DNA utilizing its
double HNH and RuvC/RNaseH-like endonuclease spaces. Basepairing between the PAM
and the phage genome is required in sort II frameworks. Be that as it may, the PAM
is perceived on an indistinguishable strand from the crRNA (the inverse strand to sort I
frameworks).
                    </p>
                    <p>
                        *Sort III frameworks, similar to sort I require six or seven Cas proteins authoritative to
crRNAs.[109][110] The sort III frameworks investigated from S. solfataricus and P. furiosus
both focus on the mRNA of phages as opposed to phage DNA genome,[61][110] which may
make these frameworks interestingly fit for focusing on RNA-based phage genomes.[60]
                    </p>
                    <p>
                        *The instrument for separating self from outside DNA amid obstruction is incorporated with
the crRNAs and is consequently likely regular to each of the three frameworks. All through
the unmistakable development procedure of each significant sort, all crRNAs contain a spacer
succession and some segment of the rehash at one or both closures. It is the incomplete rehash
arrangement that keeps the CRISPR-Cas framework from focusing on the chromosome as
base blending past the spacer grouping signals self and avoids DNA cleavage.[111] RNA-
guided CRISPR catalysts are named sort V confinement chemicals.
                    </p>
                </div>
               
                <br />
                <br />
            </div>
        );
    }
}
module.exports = introduction_to_crispr;
